core/src/nl/tudelft/watchdog/core/logic/ui/listeners/staticanalysis/StaticAnalysisMessageClassifier.java

```diff
@@ -0,0 +1,31 @@
+package nl.tudelft.watchdog.core.logic.ui.listeners.staticanalysis;
+
+/**
+ * Holder for all {@link ClassificationBundle} that can contain messages.
```

Comment:
Could you indicate the plural like so? `all {@link ClassificationBundle}s`. This is also true for other Javadoc comments. would be nice if you could have a look at them, too. Thx!